Pipfile

```diff
@@ -24,3 +24,5 @@ netCDF4 = "==1.5.4"
 powersimdata = {editable = true, path = "../PowerSimData"}
 nrel-pysam = "~=2.2"
 pyproj = "==2.6.1.post1"
+networkx = "==2.5"
+haversine = "==2.3.0"
```

Comment:
No need to install this package. You can use the `haversine` function in the `powersimdata.utility.distance` module

Reply:
Interesting to know this. Make sense to use the one from powersimdata directly.